### PR TITLE
Fix set gpu for 3D buffers/views

### PR DESCRIPTION
**Pull request by erikzenker**

### CODE REVIEW

File: include/alpaka/mem/buf/cuda/Set.hpp [hunk 1]
```diff
@@ -470,14 +474,14 @@ namespace alpaka
...
+                            dstPitchBytesX,
+                            dstWidth * sizeof(Elem),
+                            dstPitchBytesY/dstPitchBytesX));
 
                     cudaExtent const cudaExtentVal(
                         make_cudaExtent(
-                            extentWidth,
-                            extentHeight,
+                            extentWidth * sizeof(Elem),
+                            extentHeight ,
```
Comment (BenjaminW3) [inline include/alpaka/mem/buf/cuda/Set.hpp]: What is the space doing there? :smile:

**Comment by BenjaminW3** [comment]
Have you tested this with a 3 dimensional buffer? The integration tests currently only have 1D and 2D tests.

**Comment by erikzenker** [comment]
I have some simple test cases for 1-3 dim copy and set, but i would like to improve them before I add them to alpaka.

**Comment by erikzenker** [comment]
PS: I removed the space ^^

**Pull request merged**